docs/std-str.md

```diff
@@ -0,0 +1,175 @@
+# str library
+
+str library contains functions that are used for string manipulations.
+
+## concat
+
+`//.str.concat` concatenates a list of strings. It takes one argument which is
+a list string. It returns a string.
+
+
+### examples
+
+```text
+//.str.concat(["ba", "na", "na"])
+
+"banana"
+```
```

Comment:
```suggestion
- `//.str.concat(["ba", "na", "na"]) = "banana"`
```

Reply:
Use the same pattern everywhere.

Reply:
Alternatively, use tables:

| expression | equals |
|:-|:-|
| `//.str.concat(["ba", "na", "na"])` | `"banana"` |

```
| expression | equals |
|:-|:-|
| `//.str.concat(["ba", "na", "na"])` | `"banana"` |
```